docs/python/debugging.md

```diff
@@ -78,7 +78,9 @@ For information about creating and using debugging configurations see the [Initi
 The debugger can also be run from the command line. The debugger command line syntax is as follows:
 
 ```bash
-python -m debugpy --listen [<address>:]<port>
+python -m debugpy
+    --listen | --connect
+    [<address>:]<port>
```

Comment:
This should be `[<host>:]<port>` - we've already tweaked debugpy health output accordingly. 

(address would be the whole thing, host+port - I mislabeled it in help originally)

Reply:
Gotcha, updated in latest commit.